## This is the contents of an FFI database
## Related pkg: eris
##
## Author: Ferdian Pratama

string[] known_objects
string[] known_tags